sbt-mode-comint.el

```diff
@@ -184,14 +184,14 @@ what `sbt:move-marker-before-prompt-filter` did."
 (defun sbt:scala-escape-string (str)
   (mapconcat 'sbt:scala-escape-char str ""))
 
-(defconst sbt:completions-regex "^\\[completions\\] \\(.*\\)
?$")
+(defconst sbt:completions-regex "^\\[completions\\] \\(.*\\)
```

Comment:
It must be a bug in the original code, no kind of linefeed is actually needed there. I'll update the PR